content/docs/user-guide/data-management/track-sync-version.md

```diff
@@ -0,0 +1,179 @@
+        deleted: model.pkl
+```
+
+<admon type="tip">
+
+Other commands partially related to tracking data: `dvc status`,
+`dvc unprotect`, `dvc list`, `dvc import`, `dvc import-url`.
+
+</admon>
+
+[similar to git]:
+  https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository
+[links it]: /doc/user-guide/data-management/large-dataset-optimization
+[more details]: /doc/user-guide/how-to/stop-tracking-data
+
+## Synchronizing data
+
+DVC lets you [codify your data], configure the project's storage location(s),
+and stop worrying about low-level operations like copying, moving, renaming,
+uploading and downloading, etc.
+
+At a minimum, you'll have two locations: your <abbr>workspace</abbr> and a
+[cache directory]. The [data tracking](#tracking-data) operations already keep
+these in sync most of the time.
+
+<admon type="tip">
+
+The `dvc commit` and `dvc checkout` plumbing commands let you force-sync them.
+This can be useful when unexpected errors occur (e.g. cache corruption).
+
+</admon>
+
+To enable additional storage locations, you can configure [DVC remotes] with the
+`dvc remote add` and `dvc remote modify` commands (see `dvc remote` for more
+options). Once this is done, use the `dvc push` and `dvc pull` commands to
+transfer data between the project and remote storage. This is the main mechanism
+to share and back up your work.
+
+![Sync ops among locations](/img/sync-ops-locations.png) _Data sync operations
+among locations_
+
+<admon type="tip">
+
+`dvc fetch` transfers files downstream halfway -- from remote storage to the
+<abbr>cache</abbr>. This can be useful to make sure that some data is available
+for checkout later.
+
+</admon>
+
+<admon type="info">
+
+Regardless of the name, "remotes" may be set up anywhere: local file systems,
+external devices or network locations, and remote servers or cloud platforms.
+
+</admon>
+
+A more advanced strategy is to access and synchronize data assets one way, from
+misc. locations or other DVC projects (e.g. [data registry] pattern).
+`dvc list`, `dvc import` or `dvc import-url`, and `dvc update` are the main
+commands related to this.
+
+[codify your data]: /doc/use-cases/versioning-data-and-models
+[cache directory]: /doc/user-guide/data-management#the-data-cache
+[protected]: /doc/command-reference/unprotect
+[dvc remotes]: /doc/user-guide/data-management#remote-storage
+[data registry]: /doc/use-cases/data-registry
+
+## Versioning data
+
+You may have noticed that most of the tracking and synchronization commands give
+out hints about `git` commands to follow DVC operations. That's because the
+unifying aspect across DVC features (for data management and beyond) is [data
+versioning].
+
+The way this looks is that many DVC operations write small [metafiles] to the
+<abbr>workspace</abbr>, which you can in turn track and version with Git, a
+feature-rich and battle-tested [SCM] tool.
+
+![Versioning flow](/img/flow.png) _Data versioning flow on top of Git_
```

Comment:
Found this image unused already in the repo! Pretty close to what we need here:

![image](https://user-images.githubusercontent.com/1477535/197028406-95a08950-3e9d-4c9d-b77d-5ef549074c15.png)

> p.s. we should probably prune unused assets from [static/](https://github.com/iterative/dvc.org/tree/main/static) ?